src/client/visualizations/heat-map/heatmap-legend.tsx

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2017-2018 Allegro.pl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as React from "react";
+import { ConcreteSeries } from "../../../common/models/series/concrete-series";
+import "./heatmap-legend.scss";
+import { ColorScale } from "./utils/scales";
+
+interface HeatmapLegendProps {
+  scale: ColorScale;
+  series: ConcreteSeries;
+  height: number;
+}
+
+export const HeatmapLegend: React.SFC<HeatmapLegendProps> = ({ height, series, scale }) => {
+  const [min, max] = scale.domain();
+  if (isNaN(min) || isNaN(max)) return null;
+
+  const [startColor, endColor] = scale.range();
+
+  const format = series.formatter();
+  const [y1, y2] = [10, height - 10];
```

Comment:
Move `10` to some descriptively-named constant.

Reply:
`ten`?